### PR TITLE
Fix column ordering in keyboard navigation

### LINKED ISSUE #1193: Keyboard navigation does not move properly with the Right Arrow key
Live demo: https://jsfiddle.net/6r5htd05/113/

Steps to reproduce:
 
- focus the cell in the first column
- try to navigate with the Right Arrow key

### PR DESCRIPTION
**Pull request by platosha**
Fixes #1193

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-grid/1201)

### CODE REVIEW

File: src/vaadin-grid-keyboard-navigation-mixin.html [hunk 1]
```diff
@@ -242,7 +242,8 @@
       // original _focusedColumnOrder in the sorted array of orders
       // of the visible columns on the destination row.
       const dstColumns = this._getColumns(activeRowGroup, dstRowIndex);
-      const dstSortedColumnOrders = dstColumns.filter(c => !c.hidden).map(c => c._order).sort();
+      const dstSortedColumnOrders = dstColumns.filter(c => !c.hidden).map(c => c._order)
+        .sort((b, a) => (b - a));
```
Comment (limonte) [inline src/vaadin-grid-keyboard-navigation-mixin.html]: :heart: js

**Comment by limonte** [comment]
:lgtm:

After merging to master, please apply the same change to `4.x`. Thanks!

---

Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/1201#-:-L3maKUR6C7FBBVQJKFu:b-pemf2h)*

**Comment by tomivirkki** [comment]
:lgtm:

---

Reviewed 2 of 2 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/1201#-:-L3maqcDAgSQbDxzDQ9D:bnfp4nl)*

**Comment by tomivirkki** [comment]
*[src/vaadin-grid-keyboard-navigation-mixin.html, line 246 at r1](https://reviewable.io:443/reviews/vaadin/vaadin-grid/1201#-L3mZDeV4yvfW7SMfyrR-r1-246:-L3mbODyieFRb3fcTWzn:bxknle7) ([raw file](https://github.com/vaadin/vaadin-grid/blob/13fd6b6fb3e6fdb416d8d8780dfb844b0e4c464f/src/vaadin-grid-keyboard-navigation-mixin.html#L246)):*
<details><summary><i>Previously, limonte (Limon Monte) wrote…</i></summary><blockquote>

:heart: js
</blockquote></details>

^ This comment is blocking

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-grid/1201)*

**Comment by limonte** [comment]
> ^ This comment is blocking

<3 Reviewable

**Pull request merged**